encryption.js

```diff
@@ -29,26 +29,26 @@ function generateKey(passphrase, appWiseSalt) {
 }
 
 function base64ToArrayBuffer(base64) {
-    var binary_string =  window.atob(base64);
-    var len = binary_string.length;
-    var bytes = new Uint8Array( len );
-    for (var i = 0; i < len; i++)        {
-        bytes[i] = binary_string.charCodeAt(i);
-    }
-    return bytes;
+  const binary_string =  window.atob(base64);
+  const len = binary_string.length;
+  const bytes = new Uint8Array( len );
```

Comment:
Fixed! ESLint has a `space-in-parens` rule which let us auto-fix this in the 6 places we slipped in an extra space before/after a paren. :+1: